packages/flutter_reactive_ble/lib/src/reactive_ble.dart

```diff
@@ -104,8 +105,13 @@ class FlutterReactiveBle {
         print,
       );
 
-      ReactiveBlePlatform.instance =
-          const ReactiveBleMobilePlatformFactory().create();
+      if (Platform.isLinux) {
```

Comment:
I am actually thinking to inject a reactiveble platform (with the mobile as default). That way mobile users are not confronted with linux specific dependencies and the other way around.

Reply:
I don't know how to do this - can you point at an example / some info on what this requires?

Reply:
I did look to other plugins and the way it is currently done by you and in the library is the most common way. Let's stick with this.